docs/hugo/content/architecture-decision-records/ADR-2020-04-Code-Generation.md

```diff
@@ -2,8 +2,6 @@
 title: '2020-04: Why Code Generation?'
```

Comment:
Does this actually get rendered in the page? Just making sure it makes sense to nuke the title below.

Reply:
Different themes do different things with the frontmatter - with Book, it was used only for menus; with Docsy it's rendered in the page as well.